src/lib.rs

```diff
@@ -18,8 +18,6 @@
 // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
-//! libp2p is a modular peer-to-peer networking framework.
```

Comment:
was this also supposed to be removed?

Reply:
No worries Hannes, it happens a lot to me :sweat_smile:

Reply:
That calms me :)